kokoro/scripts/build/build_package.ps1

```diff
@@ -47,10 +47,24 @@ PACKAGE_VERSION,$env:PKG_VERSION
   Out-File -FilePath "$env:KOKORO_ARTIFACTS_DIR/custom_sponge_config.csv" -Encoding ascii
 
 Invoke-Program git submodule update --init
-Invoke-Program docker build -t $tag -f './Dockerfile.windows' .
+$artifact_registry="us-docker.pkg.dev"
+& gcloud auth configure-docker $artifact_registry
+
+$cache_location="${artifact_registry}/stackdriver-test-143416/google-cloud-ops-agent-build-cache/ops-agent-cache:windows"
+Invoke-Program docker build --cache-from="${cache_location}" \-t $tag -f './Dockerfile.windows' .
```

Comment:
the backslash in front of the `-t` looks like a debugging remnant or a typo, remove it?